src/components/signup/sagas.js

```diff
@@ -25,7 +25,7 @@ function* signupUser(action) {
         if(response.ok){
             const data = yield response.json();
             yield effects.put(signupSuccess());
-            yield effects.put(setCookie(data));
+            yield effects.put(getUserData(data));
```

Comment:
shouldn't this be `setUserData`

Reply:
the action grabs the user data only, I think get defines it better